src/coreclr/jit/llvm.cpp

```diff
@@ -1656,25 +1685,32 @@ void Llvm::storeLocalVar(GenTreeLclVar* lclVar)
 {
     if (lclVar->gtFlags & GTF_VAR_DEF)
     {
-        Value* valueRef = getGenTreeValue(lclVar->gtGetOp1());
-        assert(valueRef != nullptr);
+        Value* localValue = getGenTreeValue(lclVar->gtGetOp1());
+        assert(localValue != nullptr);
```

Comment:
Should this assert be inside `getGenTreeValue`?

Reply:
Moved.